parsers/einpark/derbypub.ts

```diff
@@ -22,9 +22,9 @@ export class DerbyPub implements IParser {
 
         doneCallback(dayMenu);
 
-        function parseDailyMenu(table) {
+        function parseDailyMenu(table: Cheerio) {
             const rows = table.find("tr");
-            rows.each((index, elem) => {
+            rows.each((index: number, elem: CheerioElement) => {
```

Comment:
do we ned to explicitly state the types here? i would expect that these would be inferred

Reply:
I think it started arguing, otherwise I would not change it... I will check what we can do with the rules here.

Reply:
Done